frontend/sw360-portlet/src/main/webapp/html/homepage/myprojects/view.jsp

```diff
@@ -29,8 +29,13 @@
 
 <script>
 
-    //This can not be document ready function as liferay definitions need to be loaded first
-    $(window).load(function () {
+    Liferay.on('allPortletsReady',
+        function () {
+            <portlet:namespace/>load();
```

Comment:
Why `<portlet:namespace/>load();` and not `load()` as above?

Reply:
It was the result of copy paste. :)
It doesn't matter now because Iall load methods are moved to the inline.